docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb

```diff
@@ -290,19 +282,31 @@
     "\n",
     "### Sorting DataFrame on Time Index\n",
-    "When creating a DataTable with a time index, Woodwork, by default, will sort the input dataframe first on the time index and then on the index, if specified. Because sorting a distributed DataFrame is a computationally expensive operation, this sorting is performed only when creating a DataTable from a pandas DataFrame. If a sorted DataFrame is needed when using a Dask or Koalas, the user should first sort the DataFrame before creating the DataTable.\n",
+    "When initializing with a time index, Woodwork, by default, will sort the input DataFrame first on the time index and then on the index, if specified. Because sorting a distributed DataFrame is a computationally expensive operation, this sorting is performed only when using a pandas DataFrame. If a sorted DataFrame is needed when using a Dask or Koalas, the user should manually sort the DataFrame as needed.\n",
     "\n",
-    "### Equality of DataTables\n",
-    "In order to avoid bringing a Dask DataFrame into memory, Woodwoork does not consider the equality of the underlying data when checking whether a DataTable made from a Dask or Koalas DataFrame is equal to another DataTable. This means that two DataTables with identical names, columns, indices, semantic tags, and LogicalTypes but different underlying data will return `True` if at least one of them uses Dask or Koalas.\n",
+    "### Equality of Woodwork DataFrames\n",
+    "In order to avoid bringing a Dask DataFrame into memory, Woodwoork does not consider the equality of the data when checking whether Woodwork Dataframe initialized from a Dask or Koalas DataFrame is equal to another Woodwork DataFrame. This means that two DataFrames with identical names, columns, indices, semantic tags, and LogicalTypes but different underlying data will be treated as equal if at least one of them uses Dask or Koalas.\n",
     "\n",
     "### LatLong Columns\n",
-    "When working with the LatLong logical type, Woodwork converts all LatLong columns to a standard format of a tuple of floats for Dask dataframes and a list of floats for Koalas dataframes. In order to do this, the data is read into memory, which may be problematic for large datatsets.\n",
+    "When working with the LatLong logical type, Woodwork converts all LatLong columns to a standard format of a tuple of floats for Dask DataFrames and a list of floats for Koalas DataFrames. In order to do this, the data is read into memory, which may be problematic for large datatsets.\n",
     "\n",
     "### Integer Column Names\n",
-    "Woodwork generally allows column names of any format that is supported by the underlying DataFrame. However, Dask DataTables do not currently support integer column names."
+    "Woodwork allows column names of any format that is supported by the DataFrame. However, Dask DataTables do not currently support integer column names.\n",
+    "\n",
+    "### Make Index\n",
```

Comment:
We should add a section about how we don't update the underlying index to match a Woodwork index in Dask and Koalas

Reply:
Added a section - I'm not sure it's super clear as is thought. It's a little hard for me to know how to clearly differentiate between the dataframe index and the Woodwork index in the text. Happy to make additional changes if you have ideas.